geth/wrapper.py

```diff
@@ -89,6 +89,8 @@ def construct_test_chain_kwargs(**overrides):
     overrides.setdefault('ipc_api', ALL_APIS)
 
     overrides.setdefault('verbosity', '5')
+    if 'shh' in overrides:
+        overrides.setdefault('shh', overrides['shh'])
```

Comment:
https://github.com/blog/2247-improving-collaboration-with-forks

Now by default when you open a PR from your fork you also give the maintainers of the target repo push access to that branch.  It's disable-able.